jobs/bump-lockfile.Jenkinsfile

```diff
@@ -77,44 +112,70 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
       done
     """)
 
-    if (!timestampOnly) {
-        stage("Fetch") {
-            // XXX: hack around subtle lockfile bug (jlebon to submit an
-            // rpm-ostree issue or patch about this)
-            shwrap("cp src/config/fedora-coreos-pool.repo{,.bak}")
-            shwrap("echo cost=500 >> src/config/fedora-coreos-pool.repo")
-            shwrap("cosa fetch --strict")
-            shwrap("cp src/config/fedora-coreos-pool.repo{.bak,}")
-        }
+    // The bulk of the work (build, test, etc) is done in the following.
+    // We only need to do that work if we have changes.
+    if (haveChanges) {
+        parallel "Fetch/Build/Test aarch64": {
+            shwrap("""
+               cat <<'EOF' > spec.spec
+job:
+  strict: true
+  miniocfgfile: ""
+recipe:
+  git_ref: testing-devel
```

Comment:
Shouldn't this be `${params.STREAM}` ?

Reply:
updated! eventually I want to change it to specify a specific commit rather than a branch but there is some underlying work in cosa (or gangplank) that needs to happen for that. Let's go with `${params.STREAM}` for now.